src/components/shell.js

```diff
@@ -8,23 +8,6 @@ define([], function () {
             } else {
                 window.open(url, target || '_blank');
             }
-
-        },
-        canExec: false,
-        exec: function (options) {
-            // options.path
-            // options.arguments
-            return Promise.reject();
-        },
-        enableFullscreen: function () {
-            if (window.NativeShell) {
-                window.NativeShell.enableFullscreen();
-            }
-        },
```

Comment:
These were only added for Android to work around a bug in Chrome. The fix is handled within Cordova now.